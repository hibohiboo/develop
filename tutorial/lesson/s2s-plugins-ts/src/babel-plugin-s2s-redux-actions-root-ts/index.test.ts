import * as path from 'path'
import * as pluginTester from 'babel-plugin-tester';
import { default as plugin } from '.';

const cwd = __dirname;//process.cwd()
const filename = path.resolve(cwd, '__fixtures__', 'src', 'actions', 'coin.ts')
const output = path.resolve(cwd, 'src', 'actions', 'index.ts')

pluginTester({
  plugin,
  snapshot: true,
  babelOptions: { filename },
  pluginOptions: { input: `${__dirname}/__fixtures__/**/*.ts`, output },
  tests: [
    {
      title: 'options',
      code: `import { createAction } from 'redux-actions';`,
    },
  ],
})