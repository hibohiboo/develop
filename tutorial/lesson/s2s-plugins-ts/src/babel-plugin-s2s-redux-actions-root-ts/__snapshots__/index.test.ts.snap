// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`s2s-redux-actions-root-ts options: options 1`] = `
"
import { createAction } from 'redux-actions';

      ↓ ↓ ↓ ↓ ↓ ↓

export * from \\"../../__fixtures__/src/actions/coin\\";
"
`;
